### ISSUE TITLE
Elm module dependency doesn't work on my project

### ISSUE THREAD

**Issue by loicknuchel** [erkal/kite#27]
Hi,
I'm discovering Kite and it's a really nice tool!
I wanted to use it to visualize my project dependencies (https://github.com/loicknuchel/schema-viz) but I got an error: `Repo not found. Is it a private repo? Use an access token!`
My project is public and I tried with a token, but without any success.
Do you know what could be the problem ?

**Comment by brandly** [comment]
[Looks like](https://github.com/erkal/kite/blob/140b1df2f738889bcf55bbae6a29d1b537c1f198/src/Generators/ElmDep.elm#L133) we always look for a `master` branch, but your project and many others use `main`! We could [ask the API for the default branch](https://stackoverflow.com/a/16501903/1614967) which is probably the most general approach.

**Comment by loicknuchel** [comment]
On nice! Thanks.
Indeed `main` branch is the new default on github so we have a lot of projets with both conventions (old and new ^^).
An easy solution could be to ask for the branch also, so we could even visualize impact of some PR on the dependency graph.
Anyway, now I got my workaround: create a `master` branch ^^

**Comment by brandly** [comment]
True. Allowing you to specify the branch would mean you could also explore the graph of some other branch before it's been merged into `main`.

Glad you found a workaround though 😄
